cypress/integration/smoke-test-bld.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('main page, boulder smoke test', () => {
+  beforeEach(() => {
+    cy.fixViewport();
+    cy.visit('http://localhost:5000/Boulder');
+    cy.waitForInitialContent();
+  });
```

Comment:
This and `smoke-test-pvd.spec.js` are almost identical. The only difference is the URL and the first test assertion.

It might be cleaner to abstract the location out and use a single test file:

```suggestion
describe('main page, smoke test', () => {
  ['Boulder', 'Providence'].forEach(location => {
    beforeEach(() => {
      cy.fixViewport();
      cy.visit(`http://localhost:5000/${location}`);
      cy.waitForInitialContent();
    });

    // etc...
```

Reply:
Was wondering how to consolidate the tests - this makes a lot of sense.

Reply:
Actually, I think you'll have to have the `describe` inside the `forEach` for the `beforeEach` to work correctly. My apologies.